includes/classes/class-ct-db-registration.php

```diff
@@ -57,7 +57,15 @@ public function check_user_permission() {
 		 */
 		public function return_login_registration_form( $atts, $content = '' ) {
 
-			if( ! is_user_logged_in() ) {
+			$successful_registration = false;
+			if ( ! empty( $_POST ) && isset( $_POST['ctdb_user_email'] ) ) {
```

Comment:
No need to check `! empty( $_POST )` here. You can change the whole line to this:

`if ( ! empty( $_POST['ctdb_user_email'] ) ) {`